lib/fastifySession.js

```diff
@@ -187,6 +188,7 @@ function fastifySession (fastify, options, next) {
         return
       }
 
+      session.touch()
```

Comment:
I found a bug like case while trying to implement this option. 
The issue is this:  the `session.expiry` was always dynamically calculated at the time of incoming request using `maxage` & `Date.now()` instead of loading it from session store. 

This issue was not causing any problem since `rolling` option was enabled by default. 
But if disable `rolling`  option,  `session.expiry` value will become a useless information because, it will be always unexpired value because it is calculated dynamically for each request. 

But in overall, the system was working well due to proper session cleanup taking place in 1 second interval . 

I fixed this behavior by loading `expiry` value from session store. 

When that is done, we need to manually call `touch()` to extend the expiry of session before saving it into session store.